packages/cardhost/app/components/card-manipulator.js

```diff
@@ -155,7 +155,6 @@ export default class CardManipulator extends Component {
       this.statusMsg = `card ${this.card.name} was NOT successfully created: ${e.message}`;
       return;
     }
-    this.router.transitionTo('cards.view', this.card.name);
```

Comment:
@bagby we should still transition here--if only to the newly saved card in the schema view. otherwise we are left on the /cards/new route after the card has an ID which is problematic.

Reply:
so something like:
```js
this.router.transitionTo('cards.schema', this.card.name);
```

Reply:
Good point. Along those lines, the "Save" button on the New Card page should really say "Create Card" or something.

Reply:
There is a `card.isNew` property you can use to tell the difference between a card that is new and unsaved if you want to add that message to the button.